skcosmo/preprocessing/flexible_scaler.py

```diff
@@ -198,7 +200,7 @@ def transform(self, K, y=None):
             - np.broadcast_arrays(K, self.ref_cmean)[1]
             - rmean.reshape((K.shape[0], 1))
             + np.broadcast_arrays(K, self.ref_mean)[1]
-        )
+        ) / self.scale
```

Comment:
Why we need to add `self.scale` here?

Reply:
We need to scale the kernel to appropriate size, see the appendix of the KPCovR paper for the derivation.

Reply:
Ok, thank you!